code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1105,3 +1105,101 @@
 	id = "corazone"
 	description = "A medication used to treat pain, fever, and inflammation, along with heart attacks."
 	color = "#F5F5F5"
+
+/datum/reagent/medicine/ketrazine
+	name = "Ketrazine"
+	id = "ketrazine"
+	description = "A powerful and addictive combat stimulant, capable of healing grievous wounds and enabling the user to shrug off stuns and heavy weights by stimulating tendons and muscle groups; however the strain on the body causes severe lasting damage. Use only in life-or-death situations. Overdose is almost invariably fatal."
+	reagent_state = LIQUID
+	color = "#5F42F4"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	overdose_threshold = 20
+	addiction_threshold = 5
+
+/datum/reagent/medicine/ketrazine/on_mob_life(mob/living/M)
+	M.status_flags |= IGNORESLOWDOWN
+	M.status_flags |= GOTTAGOFAST
+	M.adjustToxLoss(-3*REM, 0)
+	M.adjustBruteLoss(-5*REM, 0)
+	M.adjustFireLoss(-5*REM, 0)
+	M.adjustOxyLoss(-5*REM, 0)
+	M.AdjustStun(-80*REM, 0)
+	M.AdjustKnockdown(-70*REM, 0)
+	M.adjustStaminaLoss(-80*REM, 0)
+	M.AdjustUnconscious(-50*REM, 0)
+	M.adjustBrainLoss(0.5*REM,0)
+	switch(current_cycle)
+		if(13)
```

Comment:
should be lower than 13 probably, or you'll get this message and then next tick your muscles begin to ache terribly

Reply:
Query: if the if(n) is set to something like 1 or 2, does that mean the positive effects will stop? That was my reason for leaving the message so late.

Reply:
no, the positive effects never stop because you didn't write it that way.

Reply:
Can I just add in a (low) probability check to the messages so that they appear occasionally during each stage?

Reply:
yepp, just do if(prob(20)) or smth